docker/configs/generators/grid-settings.yml.template

```diff
@@ -0,0 +1,30 @@
+# awscli profile name, defaulted to media-service Hint cat ~/.aws/credentials
```

Comment:
yaml file replaces [ini](https://github.com/guardian/grid/blob/master/scripts/dot-properties/settings/settings.ini.template) one because we use yaml elsewhere.